apps/service_define/lib/define/application.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Define.Application do
```

Comment:
Nitpick, but would be nice to lose the boiler plate comments here.

Reply:
removed